CHANGELOG.md

```diff
@@ -97,6 +97,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Fixed a race condition with bulk granule delete causing deleted granules to still appear in Elasticsearch. Granules removed via bulk delete should now be removed from Elasticsearch.
 - **CUMULUS-1961**
   - Fixed `activeCollections` query only returning 10 results
+- **CUMULUS-xxx**
```

Comment:
I'd just drop this. You finished the work before we even had time to create a ticket

Reply:
Abdelhak is creating a ticket now.